app/serializers/books/show_serializer.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
```

Comment:
Hmm interesting, this can be a good pattern for some projects, we try to use the same serializer in all cases, but this is a good aproach for when you need different serializers accross actions, nice job!

Reply:
Anyways take care because this probably won't work by default, since rails will look for BookSerializer, and not Books::ShowSerializer